### PR TITLE
LPS-86131 SF

### PR DESCRIPTION
**Pull request by diegonvs**

### CODE REVIEW

**Comment by diegonvs** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:test:relevant

File: modules/apps/sharing/sharing-web/src/main/java/com/liferay/sharing/web/internal/portlet/SharedWithMePortlet.java [hunk 1]
```diff
@@ -43,4 +50,20 @@
 	service = Portlet.class
 )
 public class SharedWithMePortlet extends MVCPortlet {
+
+	@Override
+	public void render(
+			RenderRequest renderRequest, RenderResponse renderResponse)
+		throws IOException, PortletException {
+
+		renderRequest.setAttribute(
```
Comment (jbalsas) [inline modules/apps/sharing/sharing-web/src/main/java/com/liferay/sharing/web/internal/portlet/SharedWithMePortlet.java]: Do we really need this @diegonvs? Seems like we're only getting the value through the display context, so this shouldn't be necessary...

Comment (jbalsas) [reply modules/apps/sharing/sharing-web/src/main/java/com/liferay/sharing/web/internal/portlet/SharedWithMePortlet.java]: Nevermind, just saw that we retrieve it from the request 😂

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/adolfopa/liferay-portal/pull/782. [See changes here.](https://github.com/jbalsas/liferay-portal/compare/c6b349e6da75~3...c6b349e6da75) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/sharing/sharing-web/src/main/java/com/liferay/sharing/web/internal/portlet/SharedWithMePortlet.java (hunk 1 above)
Comment (diegonvs) [reply modules/apps/sharing/sharing-web/src/main/java/com/liferay/sharing/web/internal/portlet/SharedWithMePortlet.java]: 😂

**Pull request closed without merging**